From 3728617f8970f8b5a06cc5f73b71633c3ae1b5db Mon Sep 17 00:00:00 2001
From: Patrick Rudolph <patrick.rudolph@9elements.com>
Date: Mon, 16 Oct 2023 12:17:59 +0200
Subject: [PATCH 12/15] configurations: sbp1: Increase fan OutLimitMin

Depending on the used fan model the minimum percent is too low and
the fans stop spinning causing them to enter failsafe mode.

Increase the minimum to 21% to prevent frequent failsafe errors in
the log.

Change-Id: If3fb44ef97e800f2044730d181da3bb1b4c70651
Signed-off-by: Patrick Rudolph <patrick.rudolph@9elements.com>
---
 configurations/sbp1_chassis.json | 48 ++++++++++++++++----------------
 1 file changed, 24 insertions(+), 24 deletions(-)

diff --git a/configurations/sbp1_chassis.json b/configurations/sbp1_chassis.json
index 9a9e3df..e9e84bc 100644
--- a/configurations/sbp1_chassis.json
+++ b/configurations/sbp1_chassis.json
@@ -925,7 +925,7 @@
             "Name": "Fan01_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan01_Pwm"
             ],
@@ -952,7 +952,7 @@
             "Name": "Fan02_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan02_Pwm"
             ],
@@ -979,7 +979,7 @@
             "Name": "Fan03_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan03_Pwm"
             ],
@@ -1006,7 +1006,7 @@
             "Name": "Fan04_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan04_Pwm"
             ],
@@ -1033,7 +1033,7 @@
             "Name": "Fan05_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan05_Pwm"
             ],
@@ -1060,7 +1060,7 @@
             "Name": "Fan06_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan06_Pwm"
             ],
@@ -1087,7 +1087,7 @@
             "Name": "Fan07_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan07_Pwm"
             ],
@@ -1114,7 +1114,7 @@
             "Name": "Fan08_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan08_Pwm"
             ],
@@ -1141,7 +1141,7 @@
             "Name": "Fan09_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan09_Pwm"
             ],
@@ -1168,7 +1168,7 @@
             "Name": "Fan10_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan10_Pwm"
             ],
@@ -1195,7 +1195,7 @@
             "Name": "Fan11_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan11_Pwm"
             ],
@@ -1222,7 +1222,7 @@
             "Name": "Fan12_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan12_Pwm"
             ],
@@ -1249,7 +1249,7 @@
             "Name": "Fan13_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan13_Pwm"
             ],
@@ -1276,7 +1276,7 @@
             "Name": "Fan14_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan14_Pwm"
             ],
@@ -1303,7 +1303,7 @@
             "Name": "Fan15_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan15_Pwm"
             ],
@@ -1330,7 +1330,7 @@
             "Name": "Fan16_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan16_Pwm"
             ],
@@ -1357,7 +1357,7 @@
             "Name": "Fan17_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan17_Pwm"
             ],
@@ -1384,7 +1384,7 @@
             "Name": "Fan18_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan18_Pwm"
             ],
@@ -1411,7 +1411,7 @@
             "Name": "Fan19_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan19_Pwm"
             ],
@@ -1438,7 +1438,7 @@
             "Name": "Fan20_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan20_Pwm"
             ],
@@ -1465,7 +1465,7 @@
             "Name": "Fan21_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan21_Pwm"
             ],
@@ -1492,7 +1492,7 @@
             "Name": "Fan22_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan22_Pwm"
             ],
@@ -1519,7 +1519,7 @@
             "Name": "Fan23_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan23_Pwm"
             ],
@@ -1546,7 +1546,7 @@
             "Name": "Fan24_PID",
             "NegativeHysteresis": 0,
             "OutLimitMax": 100,
-            "OutLimitMin": 18,
+            "OutLimitMin": 21,
             "Outputs": [
                 "Fan24_Pwm"
             ],
-- 
2.41.0

